lib/shopify.mjs

```diff
@@ -1,11 +1,33 @@
 import dotenv from 'dotenv';
-import { post } from './request.mjs';
+import { get, post, put } from './request.mjs';
 import getLogger from './logger.mjs';
 
 dotenv.config();
 
 const logger = getLogger('lib/shopify');
 
+function getShopifyEndpoint(endpoint) {
+  const key = process.env.SHOPIFY_API_KEY;
+  const secret = process.env.SHOPIFY_API_SECRET;
+  const domain = process.env.SHOPIFY_URI;
+  const uri = `https://${key}:${secret}@${domain}${endpoint}`;
+  return uri;
```

Comment:
Maybe just return the uri instead of assigning?

Reply:
Yeah, good call.